<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="DisablingListWidget" name="listFrames">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pteNotes"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="FigureWidget" name="plotMain"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QComboBox" name="cbTRN">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Total</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rephasing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-Rephasing</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QSpinBox" name="iInterpolation1">
         <property name="toolTip">
          <string>Interpolation of ω₁ axis.</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Interpolation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="bGlobalColors">
         <property name="text">
          <string>Global</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QSpinBox" name="iInterpolation3">
         <property name="toolTip">
          <string>Interpolation of ω₃ axis.</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cbPAIR">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imag</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ampl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="bFixColors">
         <property name="text">
          <string>Fix</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ROI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ROIInputList" name="listROIInput" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Positive frequency gives exp(-t2*(t+1j*w)), negative gives exp(-t2*(t-1j*w))&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Central line kinetic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbKinetic"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="FigureWidget" name="plotKinetic"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Central line spectral</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSpectral"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="FigureWidget" name="plotSpectral"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbDisHorizontal">
         <property name="styleSheet">
          <string notr="true">color: #0000ff;</string>
         </property>
         <property name="text">
          <string>Horizontal</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbDisVertical">
         <property name="styleSheet">
          <string notr="true">color: #d50000;</string>
         </property>
         <property name="text">
          <string>Vertical</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbDisDiagonal">
         <property name="styleSheet">
          <string notr="true">color: #007c00;</string>
         </property>
         <property name="text">
          <string>Diagonal</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cbDisDiagonalFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Do not fit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gaussian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lorentzian</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labDisDiagonalFit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>μ: Position of peak relative to the tracer position
A: Peak maximum
FWHM: Full Width at Half Maximum</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbDisAntiDiagonal">
         <property name="styleSheet">
          <string notr="true">color: #7100aa;</string>
         </property>
         <property name="text">
          <string>Anti-Diagonal</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cbDisAntiDiagonalFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Do not fit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gaussian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lorentzian</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labDisAntiDiagonalFit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>μ: Position of peak relative to the tracer position
A: Peak maximum
FWHM: Full Width at Half Maximum</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisablingListWidget</class>
   <extends>QFrame</extends>
   <header>common/widgets/DisablingListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FigureWidget</class>
   <extends>QGraphicsView</extends>
   <header>pqr/pqr2</header>
  </customwidget>
  <customwidget>
   <class>ROIInputList</class>
   <extends>QWidget</extends>
   <header>analysis/tabCentralLines</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
